lib/go-parser/go-binary.ts

```diff
@@ -43,7 +43,7 @@ export class GoBinary {
           await eventLoopSpinner.spin();
         }
 
-        const version = module.snykNormalisedVersion();
+        const version = module.version;
```

Comment:
Nit: can we just get rid of the variable completely? We only use it twice, and I think `module.version` is clEarer about what is going on as well. (The var was introduced because we didn't want to call the function twice).

Reply:
I've inlined the version variable.